src/components/scenes/Fio/FioDomainRegisterSelectWalletScene.tsx

```diff
@@ -73,8 +73,10 @@ class FioDomainRegisterSelectWallet extends React.PureComponent<Props, LocalStat
 
   getRegInfo = async () => {
     this.setState({ loading: true })
-    const { fioPlugin, fioDisplayDenomination, route } = this.props
-    const { fioDomain, selectedWallet } = route.params
+    const { account, fioPlugin, fioDisplayDenomination, route } = this.props
+    const { fioDomain, selectedWalletId } = route.params
+    const selectedWallet = account.currencyWallets[selectedWalletId]
```

Comment:
Please call the route parameter `walletId`, and then use the `withWallet` HOC to do the lookup. Otherwise, QA is going to submit a "crash on FIO domain registration on auto-logout" bug.

Reply:
Found similar issues across various FIO scenes, applied the same fix